ambry-store/src/main/java/com/github/ambry/store/BlobStore.java

```diff
@@ -1321,23 +1321,23 @@ public void shutdown() throws StoreException {
     shutdown(false);
   }
 
-  public List<FileInfo> getSealedLogsAndMetaDataFiles(){
```

Comment:
The return type of methods need to be change because of limited scope of File info object. We will need to change the response to protocol object using an intermediate object here. So keeping this changes here commented before the fix is applied.